packages/react-native-win32-tester/src/js/examples/PlatformColor/PlatformColorExample.win32.js

```diff
@@ -113,7 +113,10 @@ function PlatformColorsExample() {
       ];
     } else if (Platform.OS === 'android') {
       colors = [
-        {label: '?attr/colorAccent', color: PlatformColor('?attr/colorAccent')},
+        {
```

Comment:
FYI That this might fail the lint check. We changes Prettier version recently, but it has an annoying side effect where you need to restart VS Code after installing, or format on save will do the wrong thing.

Reply:
Will check it out

Reply:
It passed lint checks, so seems okay 🤷‍♀️. Not sure why it changed.